Source/UnitTests/GTMSessionFetcherFetchingTest.m

```diff
@@ -91,6 +91,40 @@ - (NSString *)docRootPath {
   NSBundle *testBundle = [NSBundle bundleForClass:[self class]];
   XCTAssertNotNil(testBundle);
 
+#if SWIFT_PACKAGE
+  // Swift Pacage Manager does not support resources distribution currently,
+  // therefore the "gettysburgaddress.txt" file must be copied manually to the test bundle.
+  static dispatch_once_t onceToken;
```

Comment:
I guess this works because SwiftPM only runs test on macOS?  Writing into the test bundle would break the signature, so that won't work for other platforms.

Reply:
> Links seems sub optimal also, I thought swiftpm was supposed to have support to directly list files for supporting legacy layouts?

Swift PM will generate an error when 2 targets share the same **path**

Reply:
Swift PM does not support running tests on real devices. I got this error when tried to run them in my device

```
Testing couldn’t begin on the device “iPhone”: Logic Testing Unavailable
Logic Testing on iOS devices is not supported. You can run logic tests on the Simulator.
```